pxf/pxf-service/src/main/java/org/apache/hawq/pxf/service/UGICache.java

```diff
@@ -244,10 +246,10 @@ void setCleaned() {
         }
 
         /**
-         * @return the number of active requests using the {@link UserGroupInformation}.
+         * @return true if the UGI is being referenced by a session, false otherwise
          */
-        int countReferences() {
-            return referenceCount.get();
+        private boolean isInUse() {
```

Comment:
All function modified should be of the same type as they are being used in a similar fashion.